packages/gil/lib/BotClient.ts

```diff
@@ -139,44 +139,38 @@ export class BotClient extends Client {
     initializeTasks(): void {
         this.tasks.forEach(async (task) => {
             // TASKS THAT NEED TO RUN IMMEDIATELY ARE EXECUTED FIRST
-            if (task.runOnStartup) await task.execute();
+            if (task.runOnStartup) await this.executeTask(task);
 
             // SET TIMEOUT WILL DETERMINE THE RIGHT TIME TO RUN THE TASK FIRST TIME AFTER STARTUP
             setTimeout(async () => {
```

Comment:
Could this not just be condensed into the setInterval call after the initial call in the lines above this?